packages/edit-site/src/components/page-actions/trash-page-menu-item.js

```diff
@@ -51,7 +51,11 @@ export default function TrashPageMenuItem( { postId, onRemove } ) {
 	}
 	return (
 		<>
-			<MenuItem onClick={ () => removePage() } isDestructive>
+			<MenuItem
+				onClick={ () => removePage() }
+				isDestructive
+				variant="secondary"
+			>
```

Comment:
Since this button has no variations, a default border is expected. Therefore, the secondary variation has been added. However, this border will be removed in the future in order to unify the style of the delete button in #52597.

![delete-page](https://github.com/WordPress/gutenberg/assets/54422211/ac2e8f8b-4280-4215-942d-4e4d9443b0cd)